### ISSUE TITLE
Infinite Recursion

### ISSUE THREAD

**Issue by bsommardahl** [derekgreer/expectedObjects#2]
I'd like to request a feature. I'm using an object in a test that has a child that refers back to the parent. This causes an infinite loop when I run the test. If there could be a way to set the number of levels of recursion, it would be very nice.

Here's an example, in case i was not clear:

``` csharp
public class Parent {
    public Child Child { get; set; }
}

public class Child{
    public Parent Parent { get; set; }
}

var parent1 = new Parent();
parent.Child = new Child{ Parent = parent };

var parent2 = new Parent();
parent.Child = new Child{ Parent = parent };

parent1.ToExpectedObject().ShouldEqual(parent2); // <--- infinite loop
```

**Comment by stocksr** [comment]
could this be released?

**Comment by derekgreer** [comment]
This has been fixed in [v1.3.1](https://github.com/derekgreer/expectedObjects/blob/master/CHANGELOG.md#131-2017-08-26).
